payments/uri-handler/src/main/java/com/appcoins/uri_handler/PaymentScreenContentProvider.kt

```diff
@@ -1,12 +1,11 @@
 package com.appcoins.uri_handler
 
 import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
-import androidx.compose.runtime.Composable
+import androidx.activity.ComponentActivity
 import com.appcoins.payments.arch.PurchaseRequest
 
 interface PaymentScreenContentProvider {
-  fun handleIntent(context: AppCompatActivity, intent: Intent?)
+  fun handleIntent(context: ComponentActivity, intent: Intent?)
 
-  val content: @Composable (PurchaseRequest?, (Boolean) -> Unit) -> Unit
+  val content: (context: ComponentActivity, purchaseRequest: PurchaseRequest?, (Boolean) -> Unit) -> Unit
```

Comment:
lets rename it to `setContent` as it is what it actually does from now on

Reply:
Renamed to setContent